### DISCUSSION TITLE
Multiple window of the resonsivelyApp

### DISCUSSION THREAD

**Discussion by basvdeijnden** [responsively-org/responsively-app#1200] [Q&A]
Hi All, 

Is it possible to have multiple windows to compare 2 sites?

Greetings,
Bas

**Comment by manojVivek** [comment]
It would be possible on Windows, but not on Mac right now. 
Which OS are you using?

**Comment by knightspore** [comment]
There is currently a workaround for this on MacOS

1. Open Responsively from your Dock
2. Open your terminal
3. Enter `/Applications/ResponsivelyApp.app/Contents/MacOS/ResponsivelyApp`
